cli/command/inspect/inspector_test.go

```diff
@@ -219,3 +226,19 @@ func TestIndentedInspectorRawElements(t *testing.T) {
 		t.Fatalf("Expected `%s`, got `%s`", expected, b.String())
 	}
 }
+
+// moby/moby#32235
+func TestTemplateInspectorRawFallbackNumber(t *testing.T) {
```

Comment:
We can't hit `tryRawInspectFallback()` directly because it is not exposed by `TemplateInspector`.